.github/workflows/build-pypi.yml

```diff
@@ -5,39 +5,38 @@ on:
     branches:
       - stable
       - develop
-      - 'release/**'
+      - "release/**"
 
   pull_request:
     branches:
       - stable
-      - 'release/**'
+      - "release/**"
 
 jobs:
-
   build:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
         python-version: ["3.8"]
     steps:
-    - uses: actions/checkout@v4
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v5
-      with:
-        python-version: ${{ matrix.python-version }}
+      - uses: actions/checkout@v4
```

Comment:
Changes just for formatting should be avoided if possible (some people will format it one way, and other a different way, and we'll end up playing pingpong...).  I don't think any of the changes in this file are significant, so leave it out of the PR please...